certify-mock-identity.properties

```diff
@@ -13,21 +13,30 @@ mosip.certify.cache.store.individual-id=true
 mosip.certify.cache.security.algorithm-name=AES/ECB/PKCS5Padding
 mosip.certify.mock.vciplugin.verification-method=${mosip.certify.authn.jwk-set-uri}
 mosip.certify.mock.authenticator.get-identity-url=https://${mosip.api.public.host}/v1/mock-identity-system/identity
+mosip.certify.mock.vciplugin.issuer.key-cert=${mosip.certify.mock.vciplugin.issuer.cert}
+mosip.certify.mock.vciplugin.ca.key-cert=${mosip.certify.mock.vciplugin.ca.cert}
+mosip.certify.integration.data-provider-plugin=MockCSVDataProviderPlugin
```

Comment:
why are we using the MockCSVDataProviderPlugin for mock identity usecase ? My understanding is you should use MockIdaDataProviderPlugin

Reply:
Currently only MockCSVDataProviderPlugin is integrated with mock-certify-plugin. MockIdaDataProviderPlugin is still a different plugin in digital-credential-repo. A bug has been raised to merge it into the mock-certify-plugin: https://mosip.atlassian.net/browse/INJICERT-643. Thats why csv properties are being used for now.

Reply:
Since MockIdaDataProviderPlugin is out of scope for current release, let certify-mock-identity.properties continue with the vci plugin as before and the readme file should also refelect based on this.

Reply:
This property is added because when mock-certify-plugin jar is build then MockCSVDataProviderPlugin class needs this config to be defined. Setting the below value will ensure that VCI implementation will be used for VC generation.
`mosip.certify.issuer=PluginIssuer`

Reply:
ok, I asked for dataprovider related changes into certify-mock-identity.properties only because, I thought he mock ida data provider plugin is in scope, but since we have not yet moved that plugin inside certify-mock-plugin project, lets use mosip.certify.issuer=PluginIssuer and keep the changes in certify-mock-identity.properties to a minimum

Reply:
Removed csv related configs as it is not needed when mosip.certify.issuer value is PluginIssuer

Reply:
mosip.certify.svg-templates, can we remove this also ?

Reply:
Removed the above key from the mock-identity properties file.